client-runtime/src/main/java/com/microsoft/rest/v2/util/FlowableUtil.java

```diff
@@ -70,6 +72,70 @@ public static boolean isFlowableByteBuffer(TypeToken entityTypeToken) {
 
         });
     }
+    private static final class CountingOperator implements FlowableOperator<ByteBuffer, ByteBuffer> {
```

Comment:
A tad strange to see `FlowableOperator` used. None of the operators are `lift` based in RxJava 2.x, adds small overhead and harder to think about. Standard is to use `compose`.

Reply:
Ah, thanks. I'll have a look and correct this.